app/views/clients/show.html.slim

```diff
@@ -65,7 +65,7 @@ h1= @client.contact.full_name
 
       tr
         td= t_attr(:created_by)
-        td= link_to @client.user.full_name, user_path(@client.user)
+        td= link_to @client.user.full_name if @client.user.profile.present?, profile_path(@client.user.profile)
```

Comment:
I dont know if that will work like this
But to catch the case the user has no profile (`user.full_name` allready does that in itself) do a url_for
```
link_to @client.user.full_name, url_for(@client.user.profile || @client.user)
```

Like this, there will be a valid link, even if the user has no profile. In that case it will just not go to profile, but to user instead.